.github/workflows/helmtagbump.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: 'broadinstitute/datarepo-helm-definitions'
-          token: ${{ secrets.HELM_REPO_TOKEN }}
+          token: ${{ secrets.BROADBOT_TOKEN}}
           path: datarepo-helm-definitions
       - name: "integration-1 find and replace"
```

Comment:
Not sure if there's a way to do this, but I think it would be great if we could have an action that did the checkout, patch and merge in one step, to avoid duplicate everywhere we do this. It looks like everything is boilerplate except for the patch step. If the patch was done as a bash script, that could be one way to do it.

Reply:
Yes, this is definitely something I'd like to work on.  Additionally, our "helmtagbump.yaml" here in the UI almost identical to our "helmtagbumper.yaml" in the API - there must be a way for us to reduce all of this duplicated code.

The easiest path forward that I see is to create a new action in datarepo-actions (like we just did for the "merger" action) that we could group some of these steps together and we could greatly reduce the number of steps and the duplicated code. 

I think there is also an option for a "[workflow template](https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization)", but I haven't figured out a good use case for this.  I also think we'd have to share it with all of the DataBiosphere org.